### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by erogol**
- gradual training for Tacotron, that leads to faster convergence.
- a better way to handle memory queue for Tacotron
- lot's of refactoring and bug fixes by @twerkmeister 
- better style (TacotronGST) and speaker embedding (experimental)
- don't use sigmoid output for Tacotron.
- updates for pytorch 1.2
- server.py updates
- updated Benchmark.ipynb for gradual training.
- installation namespace fix by @reuben 
- exporting Tacotron model with LibTorch (experimental) by @reuben 
- fixing setup.py by @reuben
- switching to RADAM

### CODE REVIEW

**Comment by erogol** [comment]
@twerkmeister and @reuben if you'd have time to give a short look at this PR, that'd be perfect.

File: layers/common_layers.py [hunk 1]
```diff
@@ -240,7 +240,7 @@ def forward(self, attention_hidden_state, inputs, processed_inputs, mask):
                 attention_hidden_state, processed_inputs)
         # apply masking
         if mask is not None:
-            attention.data.masked_fill_(1 - mask, self._mask_value)
+            attention.data.masked_fill_(torch.bitwise_not(mask), self._mask_value)
```
Comment (reuben) [inline layers/common_layers.py]: This can also just be `~mask` FWIW

**Comment by reuben** [comment]
I'm testing this with the LibTorch changes.

File: layers/common_layers.py (hunk 1 above)
Comment (erogol) [reply layers/common_layers.py]: cool! I change it.

**Pull request merged**